delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/ConsistentWrite.scala

```diff
@@ -0,0 +1,29 @@
+package ch.epfl.bluebrain.nexus.delta.sdk
+
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.delta.kernel.Mapper
+import ch.epfl.bluebrain.nexus.delta.sdk.ConsistentWrite.ConsistentWriteValue
+import ch.epfl.bluebrain.nexus.delta.sdk.ReferenceExchange.ReferenceExchangeValue
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError.ConsistentWriteFailed
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import monix.bio.IO
+
+trait ConsistentWrite {
+
+  def apply[R](res: ConsistentWriteValue[_, _])(implicit rejectionMapper: Mapper[ConsistentWriteFailed, R]): IO[R, Unit]
+}
+
+object ConsistentWrite {
+  final case class ConsistentWriteValue[A, M](
```

Comment:
Do we really need that? What's the difference with `EventExchangeValue` besides the `project`?

Reply:
Do you mean ReferenceExchangeValue?

Reply:
We need `EventExchangeValue`, because we need the split between entity graph and metadata graph. Probably renaming this class would make sense now.